src/scss/objects/_tables.scss

```diff
@@ -53,7 +54,7 @@
           th,
           td {
               padding: $table-cell-padding;
-              line-height: v.$line-height-base;
+              line-height: units.$line-height-base;
```

Comment:
I'd be tempted if this is the only reference to `$line-height-base`, just to replace the reference with the function call instead -> `line-height()`